### PR TITLE
Cookies

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Section on cookies on the HTT cache page was confusing.

### CODE REVIEW

File: sites/upsun/src/define-routes/cache.md [hunk 1]
```diff
@@ -153,48 +153,45 @@ A full list of HTTP headers is available on [Wikipedia](https://en.wikipedia.org
...
 Other cookies are ignored.
 
 ```yaml {configFile="routes"}
-routes:
-    https://{default}/:
-        # ...
-            cache:
-                enabled: true
-                cookies: ["foo"]
+https://{default}/:
```
Comment (chadwcarlson) [inline sites/upsun/src/define-routes/cache.md:170]: This all looks good to me, but we should keep the top-level `routes` key on the Upsun side, no?

Comment (AnouckColson) [reply sites/upsun/src/define-routes/cache.md:170]: Oh absolutely, bad copy-paste on my end

File: sites/upsun/src/define-routes/cache.md [hunk 2]
```diff
@@ -153,48 +153,45 @@ A full list of HTTP headers is available on [Wikipedia](https://en.wikipedia.org
...
+    # ...
+        cache:
+            enabled: true
+            cookies: ["foo"]
```
Comment (AnouckColson) [inline sites/upsun/src/define-routes/cache.md:174]:
```suggestion
routes:
    https://{default}/:
        # ...
            cache:
                enabled: true
                cookies: ["foo"]
```

File: sites/upsun/src/define-routes/cache.md [hunk 3]
```diff
@@ -170,31 +175,24 @@ routes:
...
-    https://{default}/:
-        # ...
-            cache:
-                enabled: true
-                cookies: ['/^SS?ESS/']
+https://{default}/:
+    # ...
+        cache:
+            enabled: true
+            cookies: ['/^SS?ESS/']
```
Comment (chadwcarlson) [inline sites/upsun/src/define-routes/cache.md:189]:
```suggestion
routes:
     https://{default}/:
         # ...
             cache:
                 enabled: true
                 cookies: ['/^SS?ESS/']
```

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**